PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetExtractor.h

```diff
@@ -351,7 +354,7 @@ class AliEmcalJetTree : public TNamed
     Int_t           fBuffer_NumSplittings;
 
     /// \cond CLASSIMP
-    ClassDef(AliEmcalJetTree, 16) // Jet tree class
+    ClassDef(AliEmcalJetTree, 17) // Jet tree class
```

Comment:
It looks like you didn't actually modify `AliEmcalJetTree`, so based on what I see, you don't need to increment this particular ClassDef